hedera-node/src/main/java/com/hedera/services/stream/RecordStreamObject.java

```diff
@@ -137,6 +137,7 @@ public String toString() {
                 .append("TransactionRecord", transactionRecord)
                 .append("Transaction", transaction)
                 .append("ConsensusTimestamp", consensusTimestamp)
+                .append("Sidecars", sidecars)
```

Comment:
Is there any reason to not add sidecars to equals and hashCode as well ?

Reply:
Will do in the cleanup PR. 😁